templates/nmconection-vxlan.j2

```diff
@@ -0,0 +1,21 @@
+[connection]
```

Comment:
I don't see anything in here that would support plugging it into a bridge. Would it be `master={{ item.bridge }}`?

Reply:
Good point. I removed it for simplicity when testing and forgot to add it again. Also needs `slave-type=bridge` I think.
I've added it back in now